lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -1807,6 +1809,23 @@ protected function getPartialIndexSQL(Index $index)
     }
 
     /**
+     * Adds SQL for including columns in a covering index.
+     *
+     * @param \Doctrine\DBAL\Schema\Index $index
+     *
+     * @return string
+     */
+    protected function getCoveringIndexWithColIncludesSQL(Index $index)
+    {
+        // included columns can only be used with unclustered indexes
+        if ($this->supportsCoveringIndexesWithColIncludes() && $index->hasOption('include') && is_array($index->getOption('include')) && !$index->hasFlag('clustered')) {
```

Comment:
Rewrite the conditional so that the `return ''` is within the `{}`, and the `INCLUDE` addition is the success scenario (at the end in this method)